packages/front-end/components/Experiment/EditMetricsForm.tsx

```diff
@@ -38,11 +38,16 @@ const EditMetricsForm: FC<{
   const { hasCommercialFeature } = useUser();
   const hasOverrideMetricsFeature = hasCommercialFeature("override-metrics");
 
-  const { metrics: metricDefinitions, getDatasourceById } = useDefinitions();
+  const { metrics, getDatasourceById } = useDefinitions();
   const datasource = getDatasourceById(experiment.datasource);
-  const filteredMetrics = metricDefinitions.filter(
-    (m) => m.datasource === datasource?.id
-  );
+  const filteredMetrics = metrics
+    .filter((m) => m.datasource === datasource?.id)
+    .filter(
+      (m) =>
+        experiment?.project &&
+        m?.projects?.length &&
+        m.projects.includes(experiment.project)
```

Comment:
This doesn't look right.  What if the experiment or metric isn't in a project?  This will filter out all metrics and the select box will be empty